makefiles/pseudomodules.inc.mk

```
@@ -226,6 +226,7 @@ PSEUDOMODULES += shell_cmd_gnrc_ipv6_frag_stats
 PSEUDOMODULES += shell_cmd_gnrc_ipv6_nib
 PSEUDOMODULES += shell_cmd_gnrc_ipv6_whitelist
 PSEUDOMODULES += shell_cmd_gnrc_netif
+PSEUDOMODULES += shell_cmd_gnrc_netif_lora
```

A deprecation note for `gnrc_netif_cmd` for Doxygen should also be added here (yes, that sadly means, the doc for that needs to be introduced first, but this way, future release managers can remove that module later; see newer modules below for some hints how to document pseudo modules).

---

Also it's for lorawan not lora, let's fix this discrepancy ;-):

```suggestion
PSEUDOMODULES += shell_cmd_gnrc_netif_lorawan
```

---

Oh, you actually did that in code, just here with the pseudo-module-definition you forgot :-)